.github/workflows/build.yml

```diff
@@ -37,9 +51,14 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
       - uses: coursier/cache-action@v5
+
       - name: Setup Scala
         uses: olafurpg/setup-scala@v10
         with:
           java-version: "adopt@1.8"
+
+      - name: Prepare testing database
+        run: sbt flywayMigrate
```

Comment:
You've hit the nail on the head and I couldn't agree more. On the other hand every project I've worked on so far didn't have tests for database functions. This has been a bit of a wake-up call for me. 

I've seen firsthand how tricky it can be to refactor code that hasn't been tested. So, I've decided to take a different route. I'm committed to making sure my code is tested, no matter what.

As for Flyway, I see it as a stepping stone. It's a good temporary fix as we plan on switching to Ultet when it's released.

Reply:
Understood and I really love such approach, it's just that it might be more prudent to create another PR, perhaps pointing to this one, so that it's a bit smaller. But I still really appreciate this!! Just please let's notify the rest of the team about this decision however temporary it might be, it still needs to be communicated in my opinion - I'm not sure about individual capacities for our team, i.e. I don't know who and how detailed reviews of this PR will be done

Reply:
OK will mention this during our team's meeting.